File: fix_reader.py
with open('lib/lib/src/reader.dart', 'r') as f:
    content = f.read()

content = content.replace("disclosedBlocks: disclosedIndices,", "disclosedBlocks: null,")

with open('lib/lib/src/reader.dart', 'w') as f:
    f.write(content)

File: fix_reader.dart
import 'dart:io';

void main() {
  final file = File('lib/lib/src/reader.dart');
  String content = file.readAsStringSync();
  content = content.replaceFirst(
    "final Set<int> disclosedIndices = <int>{};",
    "// final Set<int> disclosedIndices = <int>{};",
  );
  file.writeAsStringSync(content);
}
